src/Lit/Experimental.fs

```diff
@@ -0,0 +1,76 @@
+[<Experimental("This is an exploration space, anything can be added or removed in this module at any point")>]
+module Lit.Experimental
```

Comment:
I want to learn how you did actually do the Decoration stuff as well as hot reload so any experiments and ideas I have in mind I will put them right here, hope you don't mind and also if you see things we can improve and take out to the real world stuff it would be nice